web/index.js

```diff
@@ -21,6 +22,20 @@ const app = require('./app');
 const initDb = db.init;
 const initServer = promisify(app.listen, app);
 async function init () {
+  const upload_dir = process.env.IMAGE_UPLOAD_DIR;
```

Comment:
OOOOOO do we need to add this to heroku, as I don't think *I have*?

Reply:
I added it dude. not sure if that was the problem. It seems checking for existence of a directory creates it if it doesn't exist, though I wouldn't've thought that. I have left the mkdir in there just in case.

Reply:
Fair enough